packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -21,7 +28,8 @@ const region = getEnvOrFail("AWS_REGION");
 const bucketName = getEnvOrFail("MEDICAL_DOCUMENTS_BUCKET_NAME");
 // converter config
 const PDFConvertTimeout = getEnvOrFail("PDF_CONVERT_TIMEOUT_MS");
```

Comment:
nit: constants either start with lower case or are all caps. Also, could already convert it to int here and avoid the `parseInt` below